otx/algorithms/action/adapters/mmaction/__init__.py

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from .data import OTXRawframeDataset
+from .models.backbones import MoViNetBase
```

Comment:
I think it should be moved into `common`. (Currently, all of backbone are there)

Reply:
I think movient is adapted from mmaction. If this should be merged with mmcv, then action classification should merge with classification task and action detection should merge detection. If we decide to separate action task with other 2d recognition task, then we may separate backbones too

Reply:
all right. the action's backbone is put aside for now.